packages/filefeeds/src/config.ts

```diff
@@ -57,6 +209,25 @@ export interface FileFeedsParams {
   baseUrl?: string
 }
 
+export interface FileFeedsLaunchParams {
```

Comment:
Why do we have `FileFeedsLaunchParams` that are different from `FileFeedParams`?

Reply:
Iirc, `FileFeedParams` is the superset of all the params supported, and all should be supported in the constructor.

While `FileFeedsLaunchParams` is expected to be only the subset of the former and only supported on the `launch(...)` call.

However, instead of the redundant definitions, I think we could use the TS utilities:
```
type FileFeedsLaunchParams = Pick<FileFeedParams, "userJwt" | "customizationKey" | ...>
```

Wdyt, @20michaelz?